Fonts.css

```diff
@@ -0,0 +1,344 @@
+}
+.head2-header{
+  width: 300px;
+  height: 150px;
+  display: flex;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+}
+
+.head3{
+  display: flex;
+  width: auto;
+  height: 150px;
+  justify-content: flex-start;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+}
+.head3-header{
+  width: 10%;
+  float:left;
+}
+.head3-lessons{
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.head4-lessons{
+  grid-column: 4 / 5;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.head5-lessons{
+  grid-column: 5 / 6;
+  grid-row: 1/2;
+
+  width: auto;
+}
+.container-instructor{
+
+  height: 700px;
+  width: 100%;
+}
+.head1-instructor{
+  width: auto;
+  grid-column: 1 / 2;
+  grid-row: 1/2;
+height: auto;
+}
+
+.head2-instructor{
+  width: auto;
+  grid-column: 2 / 3;
+  grid-row: 1/2;
+  height: auto;
+
+}
+
+.head3-instructor{
+  display: block;
+  width: auto;
+  grid-column: 3 / 4;
+  grid-row: 1/2;
+  height: auto;
+
+}
+
+footer {
+  background-color: #777;
+  padding: 10px;
+  text-align: center;
+  color: white;
+  background-image: url(footerImage.jpg);
+}
+
+.container-footer{
+  max-width: 1140px;
+  display: flex;
+  margin-right: auto;
+  margin-left: auto;
+  box-sizing: border-box;
+  text-align: center;
+
+}
+.element-row{
+width: 100%;
+display: flex;
+box-sizing: border-box;
+text-align: center;
+}
+
+.grid1 {
+  display: grid;
+  padding-top: 20px;
+  padding-bottom: 60px;
+  padding-right: 100px;
+  padding-left: 100px;
+  grid-template-columns: 360px 360px 360px 360px;
+  justify-content: center;
+  min-height: 500px;
+  grid-gap: 80px;
+}
+.grid2 {
+  display: grid;
+  padding-top: 40px;
+  padding-bottom: 60px;
+  grid-template-columns: 330px 330px 330px;
+  justify-content: center;
+  min-height: 300px;
+  grid-gap: 120px;
+}
+.grid3 {
+  display: grid;
+  padding-top: 20px;
+  padding-bottom: 60px;
+  grid-template-columns: 800px;
+  justify-content: right;
+  min-height: 500px;
+  grid-gap: 80px;
+  width: auto;
+}
+.gridwrapper_5 {
+  display: grid;
+  grid-template-columns: 360px 360px 360px 360px;
+  justify-content: center;
+  grid-auto-flow: row dense;
+  grid-gap: 1.5em;
+}
+.box1-contact{
+
+  height: auto;
+  width: auto;
+  justify-content: center;
+}
+.gridwrapper_header {
+  display: grid;
+  grid-template-columns: 3fr 3fr 3fr;
+  justify-content: center;
+  grid-auto-flow: row dense;
+  grid-gap: 1.5em;
+}
+nav {
+  width: auto;
+font-family: "oskar";
+font-weight: bold;
+margin: 2rem;
+}
+nav ul {
+  margin: 2rem;
+}
+nav a{
+  margin: 0.25rem;
+  padding: 0.25rem;
+}
+ input, button,  select {
+display: block;
+font-family: inherit;
+font-size: 100%;
+width: 300px;
+height: 25px;
+box-sizing: border-box;
+}
+textarea{
+  width: 400px;
+  height: 100px;
+
+}
+button{
+
+  width: auto;
+  height: auto;
+
+}
+input [type=submit]{
+
+  width: 100%;
+  background-color: #777;
+}
+fieldset input{
+
+  display: inline;
+}
+
+.checkbox input:checked ~ .checkmark {
+  background-color: #777;
+}
+
+.gridwrapper_contactform{
```

Comment:
You shouldn't need grid for the form layout. Keep it simpler and just use display block/display inline-block to lay these out